src/data_type/image.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/**
+ * Abstract Image attribute class.
+ */
+module.exports = class Image {
+  constructor(content) {
+    if (new.target === Image) {
+      throw TypeError('new of abstract class Image');
```

Comment:
Could this be slightly clearer? Maybe `throw TypeError('Image is an abstract class, so cannot be instantiated');` or something similar?

Reply:
I've now reworded this